setup.py

```diff
@@ -237,12 +237,45 @@ def get_tf_flags(build_ext, cpp_flags):
         return compile_flags, link_flags
 
 
-def get_mx_flags(build_ext, cpp_flags, lib_dirs):
+def get_mx_lib_dirs():
+    import mxnet as mx
+    mx_libs = mx.libinfo.find_lib_path()
+    mx_lib_dirs = []
```

Comment:
L243-L245 can be written in a more pythonic way if you like:  mx_lib_dirs = [os.path.dirname(mx_lib) for mx_lib in mx_libs]

Reply:
done